src/components/FeatureViews/Scheduler/SchedulerUtils.js

```diff
@@ -13,6 +13,42 @@ export const BootstrapInput = withStyles(theme => ({
     input: {
         borderRadius: 20,
         position: 'relative',
+        backgroundColor: 'white',
+        border: '1px solid #ced4da',
+        fontSize: 16,
+        width: 'auto',
+        padding: '10px 26px 10px 12px',
+        transition: theme.transitions.create(['border-color', 'box-shadow']),
+        // Use the system font instead of the default Roboto font.
+        fontFamily: [
+            '-apple-system',
+            'BlinkMacSystemFont',
+            '"Segoe UI"',
+            'Roboto',
+            '"Helvetica Neue"',
+            'Arial',
+            'sans-serif',
+            '"Apple Color Emoji"',
+            '"Segoe UI Emoji"',
+            '"Segoe UI Symbol"',
+        ].join(','),
```

Comment:
Do we really need to specify all this? lol

Reply:
this may have been copy pasta'd lol